easybuild/easyblocks/generic/perlmodule.py

```diff
@@ -58,16 +58,17 @@ def install_perl_module(self):
         """Install procedure for Perl modules: using either Makefile.Pl or Build.PL."""
         # Perl modules have two possible installation procedures: using Makefile.PL and Build.PL
         # configure, build, test, install
+        unset_cmd = "unset PERL_MM_OPT PERL_MB_OPT"
         if os.path.exists('Makefile.PL'):
-            run_cmd('%s perl Makefile.PL PREFIX=%s %s' % (self.cfg['preconfigopts'], self.installdir, self.cfg['configopts']))
+            run_cmd('%s %s perl Makefile.PL PREFIX=%s %s' % (unset_cmd, self.cfg['preconfigopts'], self.installdir, self.cfg['configopts']))
```

Comment:
Line becomes too long, and this won't work especially in combination with `preconfigopts`, but you can just revert this change if you use `unset_env_vars` instead.